README.md

```diff
@@ -1,4 +1,4 @@
-![FlyBuy Documentation](readme.png)
+# FlyBuy Developer Documentation {docsify-ignore-all}
```

Comment:
Is there a hidden way to tell docsify to ignore this file? If anyone is reading the README for this repo they're now greeted with a big "{docsify-ignore-all}".